src/components/Header/Header.tsx

```diff
@@ -1,11 +1,32 @@
+import { useNavigate } from 'react-router-dom';
+import { ReactSVG } from 'react-svg';
+import leftArrow from 'assets/img/left-arrow.svg';
+
 interface Props {
   title: string;
+  back?: boolean;
 }
 
-const Header = ({ title }: Props) => (
-  <header className="Header" data-testid="Header">
-    <h1 className="Title">{title}</h1>
-  </header>
-);
+const Header = ({ title, back = false }: Props) => {
+  const navigate = useNavigate();
+
+  const goToMainPage = (): void => navigate('/');
+
+  return (
+    <header className="Header" data-testid="Header">
+      <h1 className="Title">
+        {back && (
```

Comment:
Condition that show/disable left arrow SVG, the intention of this is to only show it when the back prop is passed and ideally reuse the component